### ISSUE TITLE
Apple Mac Silicon

### ISSUE THREAD

**Issue by paulmoons** [packagesdev/savehollywood#21]
Any chance this can be released for Apple Silicon?

**Comment by 1ManRockBand** [comment]
Would be fantastic if this was re-released for Apple Silicon.  

Currently using this for display computers at tradeshows for work as it's way simpler to implement than Aerial via a MDM.  Preferences are easily scriptable as well now that I finally found the preferences file in `~/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver.x86-64/Data/Library/Preferences/ByHost/fr.whitebox.SaveHollywood.plist` (which is a different path than what is listed in some of the other forums and documentation I've found).

In the meantime, if you're getting the "You cannot use this screen saver with this version of macOS. Please contact the vendor to get a newer version of the screen saver" error (and the "Options" button is grayed out in the Screen Saver preferences in System Settings), it's most likely due to Rosetta not being installed if you're using an Apple Silicon computer.

**Comment by pvinis** [comment]
Indeed, it finally works again for macOS 26 and 27 and Apple Silicon! https://github.com/pvinis/save-bollywood, `brew install pvinis/pvinis/save-bollywood`. Enjoy!
